sempyro/hri_dcat/hri_data_service.py

```diff
@@ -168,6 +168,15 @@ class HRIDataService(DCATDataService):
         }
     )
 
+    modification_date: str = Field(
```

Comment:
What's the difference between the type hint `str` and `sempyro.LiteralField`? And if there is (assuming `LiteralField` can hold a datatype and/or language), do we need `Union[str, LiteralField]` here?